src/content/docs/apm/agents/python-agent/python-agent-api/setuserid-python-agent-api.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: set_user_id (Python agent API)
+type: apiDoc
+shortDescription: This method is used for user tracking with errors inbox.
+tags:
+  - Agents
+  - Python agent
+  - Python agent API
+metaDescription: 'Python API: Used for user tracking with errors inbox.'
+redirects:
+  - /docs/agents/python-agent/python-agent-api/setuserid-python-agent-api
+---
+
+import errorsinboxPythonSetUserID from 'images/errors-inbox_screenshot-crop_python-set-user-id.png'
+
+## Syntax [#syntax]
+
+```
+newrelic.agent.set_user_id(user_id)
+```
+
+A standardized way to identify an end user on `Transaction` events with the `set_user_id` method.
+
+This method is used for user tracking, which is a standardized way to identify an end user on `Transaction` events.
+
+## Requirements [#requirements]
+
+Python agent version 8.8.0 or higher.
+
+## Description [#description]
+
+This call is used within the context of a transaction to attach an end user to a particular transaction or error event. This allows the customer to gain insight about a particular end user.
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        Parameter
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `user_id`
+
+        _string_
+      </td>
+
+      <td>
+        Required.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+## Return values [#return-values]
+
+`None`.
+
+When successful, the API will add the user ID as an agent attribute.
+
+When unsuccessful, the API will not add `enduser.id` as an agent attribute.  A failure may occur for several reasons:
+
+* The current transaction is not enabled.
+* Nothing was provided as an input to the API.
+* Something was provided but it was not a string.
+
+## Example usage [#examples]
+
+### Set user ID inside a background task [#set-user-id-example]
+
+An example of using `set_user_id` in a background task:
+
+```
```

Comment:
If it's a background task then we don't need to create an application like this in our example. Maybe just change the wording above to be "An example of using `set_user_id` inside a simple flask app to set the user id of the transaction".